store/tikv/region_cache.go

```diff
@@ -29,17 +29,25 @@ type RegionCache struct {
 	pdClient pd.Client
 	mu       sync.RWMutex
 	// TODO: store in array and use binary search
```

Comment:
If store as `regions map[RegionVerID]*Region`, this TODO is useless. It can't store in an array.